k8s/base/config/study-server-application.yml

```diff
@@ -22,6 +22,8 @@ backing-services:
     base-uri: http://loadflow-server/
   actions-server:
     base-uri: http://actions-server/
+  filter-server:
```

Comment:
Why adding filter-server ? Or, is it here for further development in study server ?